contracts/smart-contract-wallet/paymasters/verifying/singleton/VerifyingSingletonPaymaster.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity 0.8.17;
 import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 import {ReentrancyGuard} from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import {UserOperation, UserOperationLib} from "@account-abstraction/contracts/interfaces/UserOperation.sol";
-import {BasePaymaster, IEntryPoint} from "../../BasePaymaster.sol";
```

Comment:
No need to change the named imports i think

Reply:
I had to, otherwise it won't make use of Helpers.sol imported from BasePaymaster.